apps/console/src/index.jsp

```diff
@@ -276,6 +276,38 @@
                     return applicationDomain.replace(/\/+$/, '') + getTenantPath();
                 }
 
+                /**
+                 * Construct the auth params for organization login `authorize` requets.
+                 *
+                 * @remarks This only applies to the new authz runtime.
+                 *
+                 * @returns {string} Contructed auth params.
+                 */
+                function getAuthParamsForOrganizationLogins(orginalPrams) {
+                    if (startupConfig.legacyAuthzRuntime) {
+                        return orginalPrams;
+                    }
+
+                    var authParams = Object.assign({}, orginalPrams);
```

Comment:
```suggestion
                    var authParams = Object.assign({}, originalPrams);
```

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/5052/commits/b3d51c19e30a04dd87519e863cf048d8a9e35dcc